packages/react-strict-dom/src/native/modules/ContextCustomProperties.js

```diff
@@ -7,13 +7,13 @@
  * @flow strict
  */
 
+import type { CustomProperties } from '../../types/styles';
```

Comment:
Created a new export for the the updated `CustomProperties` type. Object key values are now `mixed` since values can be objects.